components/UserInfoForm.vue

```diff
@@ -39,11 +40,22 @@ watch(
 const onSubmit = handleSubmit(async (form) => {
   loading.value = true
 
+  let excludedFields = []
+  if (!settings.value.allow_edit_email) {
+    excludedFields.push('email')
+  }
```

Comment:
Isnt the actual field called `mail` and not `email` ?

Reply:
Yes you are right. Thanks